### PR TITLE
Add new userlog channel support.

### LINKED ISSUE #124: Seperate out moderation logs from user logs in #mod-logs
**Originally posted by Thomas Petersson:**

Might be handy to be able to have them separate if we ever need to review moderation based events like a kick / ban.

Thoughts?

### PR DESCRIPTION
**Pull request by scragly**
Closes #124 

This adds the new userlog channel in the default config, the constants, and applies them to relevant places where users are the focus.

### CODE REVIEW

File: bot/cogs/modlog.py [hunk 1]
```diff
@@ -428,7 +431,8 @@ def ignore(self, event: Event, *items: int):
         await self.send_log_message(
             Icons.user_unban, Colour.blurple(),
             "User unbanned", f"{member.name}#{member.discriminator} (`{member.id}`)",
-            thumbnail=member.avatar_url_as(static_format="png")
+            thumbnail=member.avatar_url_as(static_format="png"),
+            channel_id=Channels.userlog
```
Comment (sco1) [inline bot/cogs/modlog.py]: Should this be modlog?

File: bot/cogs/modlog.py [hunk 2]
```diff
@@ -381,7 +381,8 @@ def ignore(self, event: Event, *items: int):
         await self.send_log_message(
             Icons.user_ban, Colour(Colours.soft_red),
             "User banned", f"{member.name}#{member.discriminator} (`{member.id}`)",
-            thumbnail=member.avatar_url_as(static_format="png")
+            thumbnail=member.avatar_url_as(static_format="png"),
+            channel_id=Channels.userlog
```
Comment (sco1) [inline bot/cogs/modlog.py]: Should this be modlog?

Comment (scragly) [reply bot/cogs/modlog.py]: Yes. Thanks for the correction.

File: bot/cogs/modlog.py (hunk 1 above)
Comment (scragly) [reply bot/cogs/modlog.py]: Yes. Thanks for the correction.

File: bot/cogs/modlog.py (hunk 2 above)
Comment (sco1) [inline bot/cogs/modlog.py]:
```suggestion
            channel_id=Channels.modlog
```

File: bot/cogs/modlog.py (hunk 1 above)
Comment (sco1) [inline bot/cogs/modlog.py]:
```suggestion
            channel_id=Channels.modlog
```

**Pull request merged**